test/unit/item-group.test.js

```diff
@@ -14,4 +14,19 @@ describe('ItemGroup', function () {
         });
         expect(groups.length).to.be(4);
     });
+
+    describe('.parentOf checks', function () {
+        it('should work correctly for nested collections', function () {
+            var collection = new Collection(fixtures.nestedCollectionV2);
```

Comment:
Can we not use the fixtures? They're wreaking havoc and we are supposed to get rid of them soon.. every time we add fixtures, someone or the other ends up adding new tests based on them, and then our tests get coupled.